<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PigroWindow</class>
 <widget class="QMainWindow" name="PigroWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>TTY:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QComboBox" name="cbTty">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbRefreshTty">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="PigroGuiResources.qrc">
             <normaloff>:/pigro-gui/icons/reload.png</normaloff>:/pigro-gui/icons/reload.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Pigro.ini:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="lePigroIniPath">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbOpenPigroIni">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="PigroGuiResources.qrc">
             <normaloff>:/pigro-gui/icons/open-file.png</normaloff>:/pigro-gui/icons/open-file.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>device path:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="leDevicePath">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Protocol version:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="leProtoVersion">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>State:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leState">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageInfo">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>699</width>
         <height>98</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Info:</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Chip model:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leChipModel">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>644</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pbInfo">
             <property name="text">
              <string>INFO</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageStats">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>713</width>
         <height>54</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Stats:</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="pageCheck">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>699</width>
         <height>98</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Check:</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLineEdit" name="leCheckFilePath"/>
          </item>
          <item>
           <widget class="QPushButton" name="pbOpenCheckFile">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="PigroGuiResources.qrc">
              <normaloff>:/pigro-gui/icons/save-file.png</normaloff>:/pigro-gui/icons/save-file.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>715</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pbCheckFirmware">
             <property name="text">
              <string>Check</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageRead">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>699</width>
         <height>98</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Read:</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="leReadFilePath"/>
          </item>
          <item>
           <widget class="QPushButton" name="pbOpenReadFile">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="PigroGuiResources.qrc">
              <normaloff>:/pigro-gui/icons/save-file.png</normaloff>:/pigro-gui/icons/save-file.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>83</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>715</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pbReadFirmware">
             <property name="text">
              <string>Read</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageWrite">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>713</width>
         <height>54</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Write:</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="pageErase">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>713</width>
         <height>54</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Erase:</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pteMessages">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="PigroGuiResources.qrc"/>
 </resources>
 <connections/>
</ui>
